components/call_button/call_button.jsx

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React, {PureComponent} from 'react';
+import {injectIntl} from 'react-intl';
+
+import {intlShape} from 'utils/react_intl';
+
+import MenuWrapper from 'components/widgets/menu/menu_wrapper';
+import Menu from 'components/widgets/menu/menu';
+import CameraIcon from 'components/widgets/icons/camera_icon';
+
+const customStyles = {
+    left: 'inherit',
+    right: 0,
+    bottom: '100%',
+    top: 'auto',
+};
+
+class CallButton extends PureComponent {
```

Comment:
Does this need to be a class component? IMO this is a good candidate for a Functional Component.

Reply:
(0/5) For simplicity, I would keep this as a class component, since it does have state and some (minor) behaviour. We can improve it later.
Nevertheless, if you want it to be a functional component, I can give it a try.

Reply:
We want to switch to using react hooks with functional components in our codebase and in this case the state can be managed easily with hooks. I suggest the refactor because it would avoid the refactor of another component later. But if the PR is urgent I won't block it. Just a suggestion :)

Reply:
Done. Actually the state was not being used, so it was simpler than I expected.